v8go.cc

```diff
@@ -152,9 +153,19 @@ void Init() {
   return;
 }
 
-IsolatePtr NewIsolate() {
+IsolatePtr NewIsolate(IsolateOptions options) {
   Isolate::CreateParams params;
   params.array_buffer_allocator = default_allocator;
+
+  StartupData* startup_data;
+  if (options.snapshot_blob_data != nullptr) {
```

Comment:
I think this applies here: https://stackoverflow.com/a/17772117
```suggestion
  if (options.snapshot_blob_data) {
```